### PR TITLE
Misleading statement in "no brainer"

### PR DESCRIPTION
**Pull request by random82**
I think "we don't have to talk about this" is very misleading here. If someone is good at something, let's talk, share, brag and make sure other Readifarians are aware of that.

### CODE REVIEW

**Comment by todthomson** [comment]
> No brainer: _You're great at this! We don't even need to talk about it._

My understanding has been that "we don't even need to talk about it" is for where both the retro-target and the retro-partner have both rated the person "no brainer" therefore their is nothing for them to "talk about" w.r.t. what can be improved in that area (i.e. agree and move on).

> If someone is good at something, let's talk, share, brag and make sure other Readifarians are aware of that.

I certainly encourage retro-targets to highlight their "no brainers" to their retro-partners. I also highly encourage retro-partners to highlight "no brainers" for the retro-target. I just don't think that was the original intent of "we don't even need to talk about it" in this case. ;)

**Comment by random82** [comment]
It looks like we understand it in a same way so, maybe we can clarify the intent then :)

**Comment by andrewabest** [comment]
@random82 I like the modification you've made to the first line and agree with your sentiments.
I feel
> - Maybe it would be a good idea to coach someone else in that space. 
doesn't really add value to this repo. Happy to merge the change if you remove the second line.

File: README.md [hunk 1]
```diff
@@ -47,7 +47,8 @@ Not-yet-Readifarians are welcome to contribute ideas, however we'd encourage you
 - If you're aiming for a promotion, rate yourself on both your current role and your desired role. Looking at the desired role will provide clarity about gaps you might need to fill or grow into.
   - Consider discussing this before rating with a peer or mentor, so that they can provide feedback on your goals.
 - Change the terms...
-  - No brainer: _You're great at this! We don't even need to talk about it._
+  - No brainer: _You're great at this! Let's make sure people know about it._
+    - Maybe it would be a good idea to coach someone else in that space.
```
Comment (mishrsud) [inline README.md]: As @andrewabest suggested, this line does not align with the intent of the prose here, please remove

**Comment by andrewabest** [comment]
@random82 can we get the PR updated with the requested changes please? If not we will close this one out.

**Comment by andrewabest** [comment]
Closing this one out as it hasn't progressed.

**Pull request closed without merging**